.github/workflows/expire.yml

```diff
@@ -76,9 +76,17 @@ jobs:
               // Check if the PR is an AOD request by checking the AOD file existence.
               if (files.data.find(f => (f.filename === 'iam.yaml' || f.filename === 'tool.yaml'))) {
                 // Check if the PR is within the cutoff threshold
-                const updatedAt = new Date(Date.parse(pull['updated_at']));
+                // Get last committed at time from ref branch
+                const ref = pull['head']['ref']
+                const branch = await github.rest.repos.getBranch({
+                  owner: repoOwner,
+                  repo: repoName,
+                  branch: ref,
+                })
+                const lastCommittedAt = branch['data']['commit']['commit']['committer']['date']
```

Comment:
It is expected that we need to use ['commit']['commit'] here?

Reply:
Please check branch response schema in https://docs.github.com/en/rest/branches/branches?apiVersion=2022-11-28#get-a-branch
<img width="440" alt="Screenshot 2023-08-24 at 12 25 57 PM" src="https://github.com/abcxyz/access-on-demand/assets/51539171/535e7f1c-8dca-4f9b-9ab9-6d50e19c5424">